### DISCUSSION TITLE
[Projects Beta] Show x of n tasks on issue

### DISCUSSION THREAD

**Discussion by lastmeta** [community/community#13033] [Projects and Issues]
right now we can see:
![image](https://user-images.githubusercontent.com/5642485/159077672-b0999d39-5d0a-43b5-9594-6fc051241ce7.png)

and you have to go into the issue to see how many tasks completed. so bring that up to the front with the option to hide it in settings.

![image](https://user-images.githubusercontent.com/5642485/159078473-6fee60a1-411f-4fb6-a479-8754fd6b93c3.png)

**Comment by Neudrino** [comment]
This issue is probably a **duplicate** of the most popular issue:
- :arrow_right: https://github.com/github-community/community/discussions/7575

Other (possible) duplicates are:
- https://github.com/github-community/community/discussions/16616
- https://github.com/github-community/community/discussions/13033
- https://github.com/github-community/community/discussions/16721
- https://github.com/github-community/community/discussions/12050
- https://github.com/github-community/community/discussions/15929

An **answer** from Github as of 2022-06-13 can be found at: https://github.com/github-community/community/discussions/16721#discussioncomment-2939031

**Reply by lastmeta** [reply]
apparently everyone wants this.

**Comment by taoliujun** [comment]
No task progress in issue pannel' right now.
